core/src/main/resources/hudson/model/Computer/configure.jelly

```diff
@@ -28,7 +28,15 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <l:layout permission="${it.CONFIGURE}" title="${%title(it.displayName)}">
+  <l:layout permission="${it.EXTENDED_READ}" title="${%title(it.displayName)}">
+    <j:choose>
+      <j:when test="${app.hasPermission(it.CONFIGURE)}">
```

Comment:
```suggestion
      <j:when test="${it.hasPermission(it.CONFIGURE)}">
```

Same as https://github.com/jenkinsci/jenkins/pull/4531#discussion_r422063608, this is basically https://issues.jenkins-ci.org/browse/JENKINS-61321 all over, just for agents.

Just wasted an hour or so writing tests for https://github.com/jenkinsci/matrix-auth-plugin/pull/82 (including against this change) and this bug broke my test and I couldn't figure out why.